tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
@PaulHigin, We can set and display the credentials, but the logic to find the IP address to get into the machine is frequently changing.  The recommended method is to download Appveyor's file and run it.